dart/lib/src/scope_extensions/scope_extensions.dart

```diff
@@ -0,0 +1,51 @@
+export '_io_scope_extensions.dart'
+    if (dart.library.html) '_web_scope_extensions.dart';
+
+import 'dart:typed_data';
+
+import '../../attachment.dart';
+import '../scope.dart';
+
+extension ScopeExtensions on Scope {
+  void addAttachmentBytes(
```

Comment:
There are a couple data types in Dart which can be used as `bytes`:
- `Uint8List`
- `List<int>`
- `ByteData`

I think the names of the methods should reflect which data type they take.

For Flutter we could also add a `addAttachmentFromAsset` method, which reads attachments from an [`AssetBundle`](https://api.flutter.dev/flutter/services/AssetBundle-class.html)